### ISSUE TITLE
tBTC dAapp - clarity on 0.09995 TBTC minting and 0.01 testnet BTC redemption

### ISSUE THREAD

**Issue by saltypalo** [keep-network/tbtc-dapp#271]
### Description:
This might be not a bug, but rather lack of indication in the interface:
While we mint, we're sending 0.01 testBTC to the address, after that we receive 0.009995 TBTC, other fraction 0.000005 goes (as I was assuming) as a commission. Before redemption I considered, that I will get only 0.009995 testBTC back, however, I received full amount (0.01 testBTC) to my address. 

Would be convenient either to notify the user, that he will get the full amount on redemption or fix this.

Screenshot attached:
![(131)](https://user-images.githubusercontent.com/68163853/87636357-4efd8900-c749-11ea-9f4f-88929a2ae20d.png)

**Comment by mhluongo** [comment]
I believe this is a UI issue, but need to dig in to confirm- can I get the link for that Etherscan page?

**Comment by saltypalo** [comment]
@mhluongo Yes, sure, here's the link where you can see all ERC20 txs corresponding to my wallet. 2 burns and both are 0.01 and not 0.009995
https://ropsten.etherscan.io/address/0x5d3c106b91de817c4fb5e255bf87b6cd6a697d29#tokentxns

**Comment by Shadowfiend** [comment]
Yeah, signer fees are withheld during the minting process (though signers are only paid at redemption). This does mean that minting from a deposit does not give you enough TBTC to redeem it. 

Additionally, it's also true that when you redeem you receive the full lot size back, except for any Bitcoin-side transaction fees. @Gluzman I guess worth flagging as we consider tweaks to the redemption flow in service of the new IA: I think we need to be clearer about how much BTC will be returned through the redemption process (namely, the lot size minus any Bitcoin transaction fees).
